data_manager.py

```
@@ -0,0 +1,281 @@
+        # Apply vibe attributes if available
+        if 'vibe_attributes' in df.columns:
+            vibe_attrs = vibe_config.get('attributes', [])
+            if vibe_attrs:
+                # Parse vibe attributes (assuming JSON string)
+                def check_attributes(attrs_str):
+                    if pd.isna(attrs_str):
+                        return False
+                    try:
+                        attrs = json.loads(attrs_str)
+                        return any(attrs.get(attr, False) for attr in vibe_attrs)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)